// @hashgraph/sdk/lib/Symbols.cjs
"use strict";

Object.defineProperty(exports, "__esModule", {
  value: true
});
exports.validateChecksums = exports.transactions = exports.toProtobufKey = exports.toProtobuf = exports.signerPublicKeys = exports.signedTransactions = exports.signOnDemand = exports.shouldRetry = exports.setOperatorWith = exports.setOperatorAccountId = exports.responseToBytes = exports.requireNotFrozen = exports.requireFrozen = exports.requestToBytes = exports.requestTimeout = exports.operatorAccountId = exports.operator = exports.onMakeRequest = exports.nodeAccountIds = exports.minBackoff = exports.maxBackoff = exports.maxAttempts = exports.mapStatusError = exports.mapResponseHeader = exports.mapResponse = exports.makeTransactionData = exports.makeSignedTransaction = exports.makeRequestAsync = exports.makeRequest = exports.isFrozen = exports.grpcDeadline = exports.getTransactionDataCase = exports.getScheduledTransactionBody = exports.getLogId = exports.fromProtobufTransactions = exports.fromProtobufThresholdKey = exports.fromProtobufKeyList = exports.fromProtobufKey = exports.fromProtobuf = exports.execute = exports.beforeExecute = void 0;
const toProtobuf = Symbol();
exports.toProtobuf = toProtobuf;
const makeTransactionData = Symbol();
exports.makeTransactionData = makeTransactionData;
const getTransactionDataCase = Symbol();
exports.getTransactionDataCase = getTransactionDataCase;
const getLogId = Symbol();
exports.getLogId = getLogId;
const toProtobufKey = Symbol();
exports.toProtobufKey = toProtobufKey;
const requireNotFrozen = Symbol();
exports.requireNotFrozen = requireNotFrozen;
const requireFrozen = Symbol();
exports.requireFrozen = requireFrozen;
const isFrozen = Symbol();
exports.isFrozen = isFrozen;
const beforeExecute = Symbol();
exports.beforeExecute = beforeExecute;
const makeRequestAsync = Symbol();
exports.makeRequestAsync = makeRequestAsync;
const mapStatusError = Symbol();
exports.mapStatusError = mapStatusError;
const mapResponse = Symbol();
exports.mapResponse = mapResponse;
const execute = Symbol();
exports.execute = execute;
const getScheduledTransactionBody = Symbol();
exports.getScheduledTransactionBody = getScheduledTransactionBody;
const makeSignedTransaction = Symbol();
exports.makeSignedTransaction = makeSignedTransaction;
const fromProtobufKey = Symbol();
exports.fromProtobufKey = fromProtobufKey;
const fromProtobufKeyList = Symbol();
exports.fromProtobufKeyList = fromProtobufKeyList;
const fromProtobufThresholdKey = Symbol();
exports.fromProtobufThresholdKey = fromProtobufThresholdKey;
const fromProtobufTransactions = Symbol();
exports.fromProtobufTransactions = fromProtobufTransactions;
const fromProtobuf = Symbol();
exports.fromProtobuf = fromProtobuf;
const validateChecksums = Symbol();
exports.validateChecksums = validateChecksums;
const mapResponseHeader = Symbol();
exports.mapResponseHeader = mapResponseHeader;
const onMakeRequest = Symbol();
exports.onMakeRequest = onMakeRequest;
const maxAttempts = Symbol();
exports.maxAttempts = maxAttempts;
const nodeAccountIds = Symbol();
exports.nodeAccountIds = nodeAccountIds;
const signOnDemand = Symbol();
exports.signOnDemand = signOnDemand;
const minBackoff = Symbol();
exports.minBackoff = minBackoff;
const maxBackoff = Symbol();
exports.maxBackoff = maxBackoff;
const operator = Symbol();
exports.operator = operator;
const requestTimeout = Symbol();
exports.requestTimeout = requestTimeout;
const grpcDeadline = Symbol();
exports.grpcDeadline = grpcDeadline;
const shouldRetry = Symbol();
exports.shouldRetry = shouldRetry;
const transactions = Symbol();
exports.transactions = transactions;
const signedTransactions = Symbol();
exports.signedTransactions = signedTransactions;
const signerPublicKeys = Symbol();
exports.signerPublicKeys = signerPublicKeys;
const setOperatorWith = Symbol();
exports.setOperatorWith = setOperatorWith;
const operatorAccountId = Symbol();
exports.operatorAccountId = operatorAccountId;
const setOperatorAccountId = Symbol();
exports.setOperatorAccountId = setOperatorAccountId;
const requestToBytes = Symbol();
exports.requestToBytes = requestToBytes;
const responseToBytes = Symbol();
exports.responseToBytes = responseToBytes;
const makeRequest = Symbol();
exports.makeRequest = makeRequest;
